docs/sources/next/using-k6-browser/running-browser-tests.md

```diff
@@ -144,6 +144,30 @@ To run a simple local script:
 
    {{% /admonition %}}
 
+3. Optional step: running browser tests in Docker on M1/M2 Macs
+
+   1. Make sure you’re running [the latest Docker](https://docs.docker.com/engine/install/) version.
+
+   2. Update [Rosetta](https://en.wikipedia.org/wiki/Rosetta_(software)) and export an environment variable with the following:
```

Comment:
Does exporting an env variable like this mean the user would have to do this step again in case they close the terminal?

Reply:
We don’t know the exact reason, but in our tests with different machines, it didn’t work without setting the environment variable.